.eslintrc.js

```diff
@@ -1,22 +1,26 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module'
   },
   plugins: [
+    '@typescript-eslint',
     'ember'
   ],
   extends: [
     'eslint:recommended',
-    'plugin:ember/recommended'
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:ember/recommended',
+    'prettier/@typescript-eslint',
```

Comment:
`prettier/@typescript-eslint` is brought in here, but prettier itself is not being ran. I think we either need to remove this (and do prettier separately) or add `plugin:prettier/recommended` to actually enforce prettier.

Reply:
I'll remove prettier in *this* one and add it separately.